### DISCUSSION TITLE
One more infill question :) "Partly top layer"

### DISCUSSION THREAD

**Discussion by AndersDoverud** [supermerill/SuperSlicer#2575] [Q&A] [answered]
I do not have English as a mother tongue, I have a harder time finding the right settings. So this may be a simple question, or not ... :)
Figure 1: When an infill changes to top fill, in part, while the other part continues with infill, it creates a problem.
![IMG_9615](https://user-images.githubusercontent.com/80225736/160102048-854f3a83-3a42-42a7-842a-2cef6cb2b40d.JPG)
Figure 2: The nearby top layer must partially turn in the air and sometimes curl up (red rings).
![IMG_9620](https://user-images.githubusercontent.com/80225736/160102275-6696252b-79a9-404a-8c99-2c893e819d9c.JPG)
One solution would be to allow the top layer to expand throughout that layer.
Is there a setting for it?

**Comment by supermerill** [comment, marked as answer]
infill-> anchor solid infill by Xmm -> default

you can also activate the supporting dense layer to better "grip"/support the solid fill.

**Reply by AndersDoverud** [reply]
Excellent, thank you!

**Comment by realien-OG** [comment]
I had a simlar issue on an Ender S1 running Klipper .  I tried changing the anchor Solid fill, and the support dense layers, but my issue was the speed of the first layer after the infill which is not the solid layer, It is the Bridge Infill.  The default setting of 

Print Settings -> Speed- > Bridge Speed -> Internal Bridges is 150%.  

I changed it to 100% and all is well, and I have great layers on top of the infill now..

![Speed-BridgeSpeed-internalBridges](https://user-images.githubusercontent.com/111099067/184272933-e9da2156-778a-4496-bfdd-f1db6b6ad442.png)
